routes/users.js

```diff
@@ -1,8 +1,9 @@
 const express = require('express');
-const { createUser } = require('../controllers/userC');
+const { createUser, userForgotPassword } = require('../controllers/userC');
 
 const userRoute = express.Router();
 
 userRoute.post('/register', (createUser));
+userRoute.post('/userForgotPassword', (userForgotPassword));
```

Comment:
```suggestion
userRoute.post('/forgot-password', (userForgotPassword));
```

You endpoint should not be camelCase.

Reply:
Okay, noted